### PR TITLE
Adding Logic App in Bicep + readme 

### PR DESCRIPTION
**Pull request by ro100e**
## 🛠️ Description

Adding the Waste Reduction Logic App - Automate detection of idle resources and send notifications.

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [x] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [x] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [x] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: docs/_automation/waste-reduction-logic-app/README.md
Comment (flanakin) [inline docs/_automation/waste-reduction-logic-app/README.md:1]: What's the difference between what you have here and the other PR?

File: src/logic-apps/waste-reduction/README.md
Comment (flanakin) [inline src/logic-apps/waste-reduction/README.md:1]: What are these empty files?

**Review by flanakin — CHANGES_REQUESTED** [review]

File: docs/_automation/waste-reduction-logic-app/README.md
Comment (ro100e) [reply docs/_automation/waste-reduction-logic-app/README.md:1]: Step 1 in this PR will not work for the BICEP code

File: src/logic-apps/waste-reduction/README.md
Comment (ro100e) [reply src/logic-apps/waste-reduction/README.md:1]: The files needed for the template, I figured that should all be in a sperate branch. I'll probably ended up just submitting a new PR

**Review by arthurclares — COMMENTED** [review]
Updating the feature with the latest readme

**Pull request closed without merging**